UsefullExtensionLib/usefullExtensionLibrary/src/main/java/com/nbprog/usefullextensionlibrary/StringExtension.kt

```diff
@@ -0,0 +1,19 @@
+package com.nbprog.usefullextensionlibrary
+
+val String.containsLatinLetter: Boolean
+    get() = matches(Regex(".*[A-Za-z].*"))
+
+val String.containsDigit: Boolean
+    get() = matches(Regex(".*[0-9].*"))
+
+val String.isAlphanumeric: Boolean
+    get() = matches(Regex("[A-Za-z0-9]*"))
+
+val String.hasLettersAndDigits: Boolean
+    get() = containsLatinLetter && containsDigit
+
+val String.isIntegerNumber: Boolean
+    get() = toIntOrNull() != null
```

Comment:
W przypadku zbyt długiego ciągu znaków np. "123414122131313" wynik będzie false, czy jesteś tego świadom?

Reply:
No wtedy to chyba już nie Integer